src/services/user.js

```diff
@@ -423,15 +423,25 @@ const getProfile = (username) => {
 };
 
 const getAdvisors = async (username) => {
-  let advisors;
-  advisors = await http.get(`profiles/Advisors/${username}/`);
-  return advisors;
+  return await http.get(`profiles/Advisors/${username}/`);
 };
 
-async function setAdvisors(profile) {
-  if (profile.AD_Username != null) {
-    profile.Advisors = await getAdvisors(profile.AD_Username);
+const getCliftonStrengths = async (username) => {
+  try {
+    return await http.get(`profiles/clifton/${username}/`);
+  } catch (error) {
+    console.log('Clifton strengths error:', error);
+    // TODO: currently throws an error whenever clifton strengths are missing, 
+    // should just return null or empty
   }
+};
+
+async function setAdvisors(profile) {
+  profile.Advisors = await getAdvisors(profile.AD_Username);
+}
+
+async function setCliftonStrengths(profile) {
+  profile.CliftonStrengths = await getCliftonStrengths(profile.AD_Username);
```

Comment:
Would it make more sense to have the `CliftonStrengths` property on the profile actually be the array of strengths, rather than an object with the property `Strengths` that is the array of strengths?
```suggestion
  const strengths = await getCliftonStrengths(profile.AD_Username);
  profile.CliftonStrengths = strengths.Strengths;
```